presto-main/src/test/java/com/facebook/presto/sql/planner/TestLogicalPlanner.java

```diff
@@ -267,6 +293,113 @@ private Plan plan(String sql, LogicalPlanner.Stage stage)
         }
     }
 
+    private static void assertEqualPlans(Plan actual, Plan expected)
```

Comment:
That's assuming the hand-written query gets planned in the same way, which is not necessarily true.